doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseNavigationView.md

```diff
@@ -3,31 +3,161 @@ uid: Learn.Tutorials.Navigation.Advanced.NavigationView
 ---
 # How-To: Use a NavigationView to Switch Views
 
-> [!WARNING]
-> **Work in progress -** 
->This page is currently under construction. It will be available soon. 🚧
->
-> **Have questions or feedback?**
->You can help shape the documentation for this topic by providing feedback on the Uno.Extensions [repo](https://github.com/unoplatform/uno.extensions/discussions/categories/general)
-
-> [!TIP]
-> This guide assumes you used the Uno.Extensions `dotnet new unoapp-extensions` template to create the solution. Instructions for creating an application from the template can be found [here](xref:Overview.Extensions)
+Choosing the right control for your navigation needs is important, and one common choice is the WinUI `NavigationView`. This control adapts to different screen sizes and offers a uniform top-level navigation experience. `NavigationView` is a great option for adaptive, customizable, and mobile-friendly navigation. The Uno Platform extensions for navigation provide built-in support for using `NavigationView` and `NavigationViewItem` to switch between views. This tutorial will show you how to configure a `NavigationView` for use with the navigation extensions.
 
 ## Step-by-steps
 
-### Code example
+### 1. Add XAML namespace mappings
+
+* Add the following namespace mappings to the root element of your XAML page:
+
+  ```xml
+  xmlns:uen="using:Uno.Extensions.Navigation.UI"
+  xmlns:utu="using:Uno.Toolkit.UI"
+  ```
+
+* Make sure that the project you're building depends on Uno.WinUI. Only if you're building a UWP app (Uno.UI), you will need to take an additional step. Add the following namespace mapping and prefix every reference to `NavigationView` or `NavigationViewItem` with _muxc_ for this tutorial:
+
+  ```xml
+  xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+  ```
+
+### 2. Define the view's layout
+
+* Add a `Grid` element to the root of your XAML page. This will be the container for the `NavigationView` and the content area.
+
+  ```xml
+  <Grid>
+      ...
+  </Grid>
+  ```
+
+* Add a `RowDefinition` to the `Grid`'s `RowDefinitions` collection. This will define the height of the `NavigationView`'s menu.
+
+  ```xml
+  <Grid.RowDefinitions>
+      <RowDefinition Height="Auto" />
+      <RowDefinition />
+  </Grid.RowDefinitions>
+  ```
+
+* Add the `NavigationBar` control to the `Grid`. This will be the header for the app name.
+
+  ```xml
+  <utu:NavigationBar Content="Main Page"
+                    Style="{StaticResource MaterialNavigationBarStyle}" />
+  ```
+
+* Add a `NavigationView` control to the `Grid`. This will be the menu for the app.
+
+  ```xml
+  <NavigationView Grid.Row="1">
+      ...
+  </NavigationView>
+  ```
+
+* Add a `Grid` element to the `NavigationView`. This will be the container for the content area.
```

Comment:
You should point out that the NavigationView has two panes for content, the Pane which will contain the list of navigation items and the main content area